src/Rings/groebner/factoring.jl

```diff
@@ -0,0 +1,83 @@
+@doc raw"""
+    factoring_groebner_basis(I::MPolyIdeal; ordering::MonomialOrdering = default_ordering(base_ring(I))) 
+
+Return a factorization of the Gröbner basis of `I` with respect to `ordering` which must be global.
+The intersection of these ideals has the same zero set as the input ideal, i.e.
+the radical of the intersection coincides with the radical of the input ideal.
```

Comment:
```suggestion
Return a list of Gröbner bases with respect to `ordering` which must be global.
The intersection of these ideals has the same zero set as the input ideal `I`, i.e.
the radical of the intersection coincides with the radical of `I`.
```

Minor rewording since a factorization of a Groebner basis is (to my knowledge) not a commonly used term.